text/3463-crates-io-policy-update.md

```diff
@@ -0,0 +1,233 @@
+
+# Proposal
+[guide-level-explanation]: #guide-level-explanation
+
+The following is a proposed new "Terms of Use" policy for crates.io, replacing <https://crates.io/policies> and <https://crates.io/data-access>.
+
+-------------------------------------------------------------------------------
+
+# Terms of Use
+
+**Short version:** _crates.io is a critical resource for the Rust ecosystem,
+which hosts a variety of packages from a diverse group of users. That resource
+is only effective when our users are able to work together as part of a
+community in good faith. While using crates.io, you must comply with our
+Acceptable Use Policies, which include some restrictions on content and conduct
+on crates.io related to user safety, intellectual property, privacy,
+authenticity, and other limitations. In short, be excellent to each other!_
+
+We do not allow content or activity on crates.io that:
+
+- violates the [Code of Conduct](https://www.rust-lang.org/policies/code-of-conduct)
+  of the Rust project
+- is unlawful or promotes unlawful activities, incurring legal liability in the
+  countries the Rust Foundation officially operates in
+- is libelous, defamatory, or fraudulent
+- amounts to phishing or attempted phishing
+- infringes any proprietary right of any party, including patent, trademark,
+  trade secret, copyright, right of publicity, or other right
+- unlawfully shares unauthorized product licensing keys, software for
+  generating unauthorized product licensing keys, or software for bypassing
+  checks for product licensing keys, including extension of a free license
+  beyond its trial period
+- contains malicious code, such as computer viruses, computer worms, rootkits,
+  back doors, or spyware, including content submitted for research purposes
+  (tools designed and documented explicitly to assist in security research are
+  acceptable, but exploits and malware that use the crates.io registry as a
+  deployment or delivery vector are not)
+- uses obfuscation to hide or mask functionality
+- is discriminatory toward, harasses or abuses another individual or group
+- threatens or incites violence toward any individual or group, especially on
+  the basis of who they are
+- is using crates.io as a platform for propagating abuse on other platforms
+- violates the privacy of any third party, such as by posting another person's
+  personal information without consent
+- gratuitously depicts or glorifies violence, including violent images
+- is sexually obscene or relates to sexual exploitation or abuse, including of
+  minors (see <https://docs.github.com/en/site-policy/acceptable-use-policies/github-sexually-obscene-content>)
```

Comment:
This link should probably be rehosted or directly copied in, as an official policy document being susceptible to link rot sounds like a bad idea.

Reply:
done ✅